src/drivers/osd/msp_osd/uorb_to_msp.cpp

```diff
@@ -279,6 +339,54 @@ msp_raw_gps_t construct_RAW_GPS(const sensor_gps_s &vehicle_gps_position,
 	return raw_gps;
 }
 
+msp_rendor_latitude_t construct_rendor_GPS_LAT(const sensor_gps_s &vehicle_gps_position)
+{
+	msp_rendor_latitude_t lat;
+
+	lat.screenYPosition = 0x0A; //
+	lat.screenXPosition = 0x29; //
+
+	if (vehicle_gps_position.fix_type >= 2) {
+		sprintf(&lat.str[0], "%.6f", vehicle_gps_position.latitude_deg);
+
+	} else {
+		sprintf(&lat.str[0], "%.6f", 0.0);
+	}
+
+	return lat;
+}
+
+msp_rendor_longitude_t construct_rendor_GPS_LON(const sensor_gps_s &vehicle_gps_position)
+{
+	msp_rendor_longitude_t lon;
+
+	lon.screenYPosition = 0x09; //
+	lon.screenXPosition = 0x29; //
```

Comment:
```suggestion
	lon.screenYPosition = 0x09; 
	lon.screenXPosition = 0x29; 
```
can you do another pass and clean up stray text and any other odd or out of place things? :smiley_cat:

Reply:
Done